src/App.tsx

```diff
@@ -1,24 +1,197 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Section } from './Components/Todolist';
+import { TodosFooter } from './Components/TodosFooter';
+import { client } from './utils/fetchClient';
 
-const USER_ID = 0;
+const USER_ID = 'https://mate.academy/students-api/todos?userId=11910';
 
 export const App: React.FC = () => {
+  const [inputValue, setInputValue] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState('all');
+  const [typeError, setTypeError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const LOADING = 'loading';
+  const ADD = 'add';
+  // const DELETE = 'delete';
+  // const CHANGE = 'change';
```

Comment:
Remove comments. Try it write obvious code that explains all we need without comments)
Fix all cases